atlasdb-dbkvs/build.gradle

```diff
@@ -31,6 +31,8 @@ dependencies {
   implementation project(':commons-executors')
   implementation project(':timestamp-api')
 
+  runtimeOnly 'com.oracle.database.jdbc:ojdbc11'
```

Comment:
What caused this change?

Reply:
I should remove this. We get these runtime dependencies from `atlasdb-dbkvs-hikari`, which is where we actually use these classes at runtime. AFAICT the `atlasdb-dbkvs` package itself doesn't use these classes at runtime.